### ISSUE TITLE
Support for list kwargs

### ISSUE THREAD

**Issue by kumarappan-arumugam** [amoffat/sh#529]
For ex:

```
s3cmd_options = dict(
    cache_control="public, max-age={}".format(365 * 24 * 60 * 60),
    exclude="*",
    include=[
        "bar/assets/*",
        "foo/assets/*",
    ],
)
sh.aws.s3.sync(local_path, dest_path, **s3cmd_options)
```

Expected:
`/usr/bin/aws s3 sync local_path dest_path --exclude=* --include=bar/assets/* --include=foo/assets/* --cache-control=public, max-age=31536000`

Actual:
`/usr/bin/aws s3 sync local_path dest_path --exclude=* --include=['bar/assets/*', 'foo/assets/*'] --cache-control=public, max-age=31536000`

Most of the command line args do not support array arg value (maybe space-separated string). I know this can be worked around by sending array as `sh.aws.s3.sync(local_path, dest_path, '--include="bar/assets/*"', '--include="foo/assets/*"')` but that kind of defeats the purpose of passing kwargs. Maybe there is a way to do this that I'm missing.

**Comment by amoffat** [comment]
There is no way to do that currently, but it seems like a fair request at first glance.

**Comment by amoffat** [comment]
Thanks for the feature suggestion, this is now implemented in 2.0.0
